client/src/pages/Profile/ProfileDetail/BookingCard/BookingForm/BookingForm.tsx

```diff
@@ -0,0 +1,153 @@
+import { Dispatch, SetStateAction } from 'react';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import Rating from '@mui/material/Rating';
+import TextField from '@mui/material/TextField';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import Box from '@mui/material/Box';
+import MobileDateRangePicker from '@mui/lab/MobileDateRangePicker';
+import DateRangeIcon from '@mui/icons-material/DateRange';
+import { DateRange } from '@mui/lab/DateRangePicker';
+import TimePicker from '@mui/lab/TimePicker';
```

Comment:
It might be worth grouping these imports as they get longer.

Reply:
Applied